src/validationinterface.h

```diff
@@ -34,6 +36,8 @@ class CValidationInterface {
     virtual void Inventory(const uint256 &hash) {}
     virtual void ResendWalletTransactions(int64_t nBestBlockTime) {}
     virtual void BlockChecked(const CBlock&, const CValidationState&) {}
+    virtual void GetScriptForMining(boost::shared_ptr<CReserveScript>&) {};
+    virtual void UpdateRequestCount(const CBlock&) {};
```

Comment:
Renamed `UpdateRequestCount` (name does not reflect the purpose of the method) to `ResetRequestCount`.